docs/topics/api/addons.rst

```diff
@@ -119,9 +119,9 @@ Detail
 This endpoint allows you to fetch a specific add-on by id, slug or guid.
 
     .. note::
-        Non-public add-ons, or add-ons with only unlisted versions, require
-        authentication and either reviewer permissions or a user account listed
-        as a developer of the add-on.
+        Non-public add-ons and add-ons with only unlisted versions require both:
+            * authentication
```

Comment:
Have you compiled the sphinx docs to check it likes `*` as a bullet point? 'tis a fickle beast.

Reply:
Github's auto-preview indicates it displays the bullet points but makes the preceding sentence bold, which looks weird.  Github doesn't know about our extensions and home brewed styles either.

Reply:
<img width="743" alt="screenshot 2017-09-10 17 41 37" src="https://user-images.githubusercontent.com/90871/30251022-de25d0b6-964f-11e7-9c48-7ef2845cd87c.png">

It looks like this, so seemingly good.

Reply:
Oh, I tweaked it so it's not bold.